dependencycheck.properties

```diff
@@ -1,13 +1,19 @@
 category=Integration
 description=Integrates Dependency-Check reports into SonarQube
 homepageUrl=https://github.com/dependency-check/dependency-check-sonar-plugin
-archivedVersions=2.0.2
-publicVersions=2.0.3
+archivedVersions=2.0.2,2.0.3
+publicVersions=2.0.4
 defaults.mavenGroupId=org.sonarsource.owasp
 defaults.mavenArtifactId=sonar-dependency-check-plugin
 
+2.0.4.description=Integrates Dependency-Check reports into SonarQube
```

Comment:
You can't have the same description for every release. This value tells users what changed in _this version_ so they can judge whether they want the upgrade.